static/src/javascripts/projects/common/modules/commercial/third-party-tags/plista.js

```diff
@@ -0,0 +1,98 @@
+define([
+    'fastdom',
+    'common/utils/$',
+    'common/utils/config',
+    'common/utils/detect',
+    'common/utils/mediator',
+    'common/utils/template',
+    'common/modules/identity/api',
+    'common/modules/commercial/commercial-features',
+    'text!common/views/commercial/plista.html'
+], function (
+    fastdom,
+    $,
+    config,
+    detect,
+    mediator,
+    template,
+    identity,
+    commercialFeatures,
+    plistaStr
+) {
+
+    var plistaTpl = template(plistaStr);
+    var selectors = {
+        widget: '.js-plista',
+        container: '.js-plista-container'
+    };
+
+    function loadInstantly() {
+        return !document.getElementById('dfp-ad--merchandising-high') ||
+            detect.adblockInUse();
+    }
+
+    function identityPolicy() {
+        return !(identity.isUserLoggedIn() && config.page.commentable);
+    }
+
+    function shouldServe() {
+        return commercialFeatures.outbrain &&
+                config.switches.plistaForOutbrainAu &&
+                config.page.edition == 'AU' &&
```

Comment:
You could in fact remove that line and the previous one, since you already know they are both true (see `third-party-tags`)